File: Nível 1/2339 - Aviões de Papel/main.lua
valor = io.read()
competidores,folhas,folhaPCompetidor = valor:match("(%S+)%s+(%S+)%s+(%S+)")

competidores = tonumber(competidores)
folhas = tonumber(folhas)
folhaPCompetidor = tonumber(folhaPCompetidor)

calc1 = folhas / folhaPCompetidor
calc2 = calc1 / competidores

calc3 = math.floor(calc2)

if calc3 > 0 then
    print("S")
elseif calc3 <= 0 then
    print("N")
else 
    print("erro")
end

File: Nível 1/2339 - Aviões de Papel/main.py
valor = input("A: ")

competidores,folhas,folhaPCompetidor = valor.split(" ")

competidores = int(competidores)
folhas = int(folhas)
folhaPCompetidor = int(folhaPCompetidor)

calc1 = folhas / folhaPCompetidor
calc2 = calc1 / competidores

calc3 = int(calc2)

if calc3 > 0:
    print("S")
elif calc3 <= 0:
    print("N")
else:
    print("erro")